lrc_file/LrcFile.py

```diff
@@ -192,6 +194,35 @@ def cpu_result_data(self) -> dict[str, float]:
         """
         return self._cpu_metrics
 
+    @property
+    def evaluation_results(self):
+        return [
+            result
+            for result in self._data.results
+            if isinstance(result, BaselineEvaluationResult)
+        ]
+
+    @property
+    def cpu_evaluation_data(self) -> dict[str, float]:
+        """
+            Returns CPU metrics with its values used during evaluation.
+        """
+        cpu_results = [
+            result.data["cpu"]
+            for i, result in enumerate(self._data.results)
+            if _is_cpu_measurement_result(result)
+        ]
+        evaluation_data = {}
+
+        for result in self.evaluation_results:
+            if not isinstance(result.data["comparisons"][0]["current_result"], CPUMeasurementResults):
+                continue
+
+            if (agg_cpu_res := result.data["comparisons"][0]["current_result"].utilization) in cpu_results:
+                evaluation_data[result.data["comparisons"][0]["metric_name"]] = agg_cpu_res.average
```

Comment:
i'm confused by the combination of these conditions. shouldn't this be simpler now?

1. `evaluation_results` returns all of the results that contain a baseline evaluation
2. we check that a specific evaluation result is from a cpu measurement with the `isinstance` check
3. why do we now have to check against `cpu_results`? isn't that already covered by the previous two checks? Can't we just export `metric_name -> average` to the result dictionary?

Reply:
imo this should have almost the same structure of code flow as the `flow_evaluation_data` implementation.

Reply:
yep, i'd say now that that if is not necessary at all.  
got a more generic solution tested, ready to review